src/Core/Commander/Providers/Fetcher.js

```diff
@@ -0,0 +1,41 @@
+import { TextureLoader } from 'three';
```

Comment:
> We already have way bigger dependencies (Mesh, Materials, Vectors)

It's true and normal, the use of THREE should only encapsulate in objects in Renderer folder.
I would like to have the minimum use of THREE in folders Scene, Core and Globe. It would be better in terms of design.

Ok for the moment, it's not very important, so keep textureLoader in Fetcher